ror-demo-cluster/conf/es/readonlyrest.yml

```
@@ -18,46 +18,49 @@ readonlyrest:
-    - username: user2
-      auth_key: user2:test
-      groups:
-        - id: "EndUsers"
-          name: "End Users"
+
+  - username: admin
+    auth_key: admin:admin
+    groups:
+      - id: "Administrators"
+        name: "Administrators"
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)